sycl/include/CL/sycl/detail/pi.h

```diff
@@ -126,7 +126,8 @@ typedef enum {
   PI_PLATFORM_INFO_NAME = CL_PLATFORM_NAME,
   PI_PLATFORM_INFO_PROFILE = CL_PLATFORM_PROFILE,
   PI_PLATFORM_INFO_VENDOR = CL_PLATFORM_VENDOR,
-  PI_PLATFORM_INFO_VERSION = CL_PLATFORM_VERSION
+  PI_PLATFORM_INFO_VERSION = CL_PLATFORM_VERSION,
+  PI_PLATFORM_INFO_ext_oneapi_P2P = 0x40110
```

Comment:
I'm not completely sure whether 'ext_oneapi_P2P' is the preferred name for such an extension.  I imagine that 'P2P' is not favoured since it does not indicate that it is an extension.  I can also use the capitalization, e.g. 'EXT_ONEAPI_P2P' once the preferred name is confirmed.

Reply:
We've added many device info extensions already, see https://github.com/intel/llvm/blob/4af2eb51455669767ffb1ba473a66a481c0e37f7/sycl/include/CL/sycl/detail/pi.h#L288

No naming convention, just use all capital case. The important part is to pick a value that isn't used by anything else already.